### PR TITLE
Add MapBox Dark to useMapStyle

### PR DESCRIPTION
**Pull request by juniormelo26**
Adding the dark map for those who have a MapBox key.

### CODE REVIEW

**Comment by juniormelo26** [comment]
@tananaev

File: modern/src/resources/l10n/en.json [hunk 1]
```diff
@@ -1,594 +1,595 @@
 {
-    "sharedLoading": "Loading...",
```
Comment (tananaev) [inline modern/src/resources/l10n/en.json:2]: Something happened with the localization file.

**Comment by juniormelo26** [comment]
Added line 361 "mapMapboxStreetsDark": "Mapbox Streets Dark"

**Comment by tananaev** [comment]
It looks like you reformatted the whole file. You have to fix that.

File: modern/src/resources/l10n/en.json (hunk 1 above)
Comment (juniormelo26) [reply modern/src/resources/l10n/en.json:2]: I didn't understand why it was formatted like that, I flew or corrected it.

**Comment by juniormelo26** [comment]
@tananaev  I made the correction. Please check when you can.

**Pull request merged**